config/ember-try.js

```diff
@@ -8,35 +8,34 @@ module.exports = async function () {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-3.24',
+        name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
-            'ember-source': '~3.24.3',
+            'ember-source': '~3.28.0',
           },
         },
       },
       {
-        name: 'ember-lts-3.28',
+        name: 'ember-4.0',
         npm: {
           devDependencies: {
-            'ember-source': '~3.28.0',
+            'ember-source': '~4.0.0',
           },
         },
       },
       {
-        name: 'ember-4.0',
+        name: 'ember-lts-4.4',
```

Comment:
Can we go ahead with adding an `ember-try` scenario for 4.8 LTS?

Reply:
We don't need to add 4.8 as it's the default (in package.json) and tested via the normal tests. It seems `ember-cli-update` takes good care of that and the next update to ember-cli v4.12 will add the `ember-lts-4.8` scenario.

I don't get the comment about the branch protection. Can you explain that?

Reply:
Ah, good point. Then yeah, we can add `ember-lts-4.8` later when you update `ember-cli` to `4.12`.

By branch protection, I was thinking of https://github.com/ember-intl/cp-validations/settings/branches. I'm not sure if you have permission to access this page.

<img width="1440" alt="" src="https://github.com/ember-intl/cp-validations/assets/16869656/9b289525-1dc2-4093-b295-26956a23733d">

At the moment, because there are no branch protections, pull requests (PRs) are allowed to be merged even if CI fails. In general, it'd be a good idea to set up a branch protection so that a PR can be merged only if CI passes.

Reply:
Got it! I can not see the page. If you want you can turn on the branch protection but don't be too restrictive.

Reply:
Sounds good. I enabled these options:

- [x] Require a pull request before merging
    - When enabled, all commits must be made to a non-protected branch and submitted via a pull request before they can be merged into a branch that matches this rule.
- [x] Require status checks to pass before merging (`embroider-optimized` isn't considered for now)
    - Choose which status checks must pass before branches can be merged into a branch that matches this rule. When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed.
- [x] Require signed commits
    - Commits pushed to matching branches must have verified signatures.